src/task-runtime.ts

```diff
@@ -150,7 +150,10 @@ class RunTask {
         let command = "";
         let hasError = false;
         const cmd = exec.split(" ")[0];
-        if (platform() == "win32" && ["mkdir", "mv", "rm"].includes(cmd)) {
+        if (
+          platform() == "win32" &&
+          ["mkdir", "mv", "rm", "cp", "cd"].includes(cmd)
```

Comment:
This is great addition, but I don't think `cd` is supported by `shx`. [1]

```suggestion
          ["mkdir", "mv", "rm", "cp"].includes(cmd)
```

[1] https://github.com/shelljs/shx#unsupported-commands

Reply:
Thanks. I misread the documentation.